src/Compilers/CSharp/Test/Emit/Emit/CompilationEmitTests.cs

```diff
@@ -3702,7 +3702,7 @@ public void IllegalNameOverride()
             compilation.VerifyDiagnostics();
 
             var result = compilation.Emit(new MemoryStream(), options: new EmitOptions(outputNameOverride: "x\0x"));
-            result.Diagnostics.Verify(
+            result.Diagnostics.Verify(fallbackToErrorCodeOnlyForNonEnglish: true,
```

Comment:
>Verify [](start = 31, length = 6)

VerifyWithFallbackToErrorCodeOnlyForNonEnglish? Could you please explain what happens with this test in its original form?